tests/factories/__init__.py

```diff
@@ -0,0 +1,27 @@
+import factory
+
+
+class PlayerFactory(factory.django.DjangoModelFactory):
+
+    nickname = factory.Faker("name")
+    motto = ""
+
+    class Meta:
+        model = "mars_api.Player"
+
+
+class GameFactory(factory.django.DjangoModelFactory):
+    game_map = "Tharsis"
+
+    class Meta:
+        model = "mars_api.Game"
+
+
+class PlayerScoreFactory(factory.django.DjangoModelFactory):
+
```

Comment:
Consistency with new lines - if you don't have a docstring then there's no need for one. If you do, then a newline after the docstring should be there

Reply:
I've just noticed the structure:
```
tests/factories
├── __init__.py
└── dictionaries.py
```

This doesn't really make sense - you should either have it all in `__init__.py` or leave it empty and create another `.py` file for your model factories. I'd just go with transforming the folder to a file:
```
tests/factories.py
```